lightgbm-4.0.0/recipe/meta.yaml

```diff
@@ -12,12 +12,11 @@ source:
   git_tag: v{{ version }}
   patches:
     # 02xx - GPU only patch specific to open-ce (maybe)
-    - 0201-Build-fix-for-cuda.patch                         #[build_type == 'cuda']
```

Comment:
Are these patches no longer required?

Reply:
Not needed during build time of lightgbm, Verified build on PPC and x86 platform too.

Reply:
We've also removed patch for cuda compute capabilities. I hope we are using right compute inspite of that. Please verify it once, if not done already.